mp/src/game/shared/neo/neo_gamerules.cpp

```diff
@@ -1333,8 +1333,8 @@ bool CNEORules::RoundIsMatchPoint() const
 		{
 			const int roundDiff = neo_round_limit.GetInt() - (m_iRoundNumber - 1);
 			return (!RoundIsInSuddenDeath() &&
-				((teamJinrai->GetRoundsWon() + 1) > (teamNSF->GetRoundsWon() + roundDiff)) ||
-				((teamNSF->GetRoundsWon() + 1) > (teamJinrai->GetRoundsWon() + roundDiff)));
+				(((teamJinrai->GetRoundsWon() + 1) > (teamNSF->GetRoundsWon() + roundDiff)) ||
```

Comment:
This seems like an implementation change unrelated to the refactor. But I guess we can merge it here if you think this should be changed?

Reply:
Upon further pondering, I guess this doesn't change the truthness after all? It is a bit confusing with all the braces though.

Reply:
Perhaps we could open this up as something like:
```cpp
			if (RoundIsInSuddenDeath())
			{
				return false;
			}

			if (teamJinrai->GetRoundsWon() + 1 > teamNSF->GetRoundsWon() + roundDiff)
			{
				return true;
			}

			if (teamNSF->GetRoundsWon() + 1 > teamJinrai->GetRoundsWon() + roundDiff)
			{
				return true;
			}

			return false;
```
so it's a bit more readable?

Reply:
Did it a bit different style but done nevertheless.